src/gc/impl/conservative/gc.d

```diff
@@ -2538,6 +2538,10 @@ struct Pool
     //  only for the first and the last page.
     uint* bPageOffsets;
 
+    // The small object pool uses the same array to keep a chain of pages with the same
+    // bin size and free pages (searchStart is first free page)
```

Comment:
This comment is a bit premature, but keeping track of pages with the same bin size will be needed for deferring incremental `recover` into later allocations.